### ISSUE TITLE
Package deb error on reinstall/upgrade

### ISSUE THREAD

**Issue by Vovan-VE** [centrifugal/centrifugo#219]
I have already installed centrifugo deb package. Upgrade caused by new release or manual reinstall for the package cause an error, probably by the line:
https://github.com/centrifugal/centrifugo/blob/ba961d3b50c6c95533b6d61629dd29592ad186c0/extras/packaging/post_install.sh#L20
This event switches package status to `iF` (install, half-configured). All subsequent system uprgades are trying to reconfigure the package again and hits the same error.

Here is some output for this:
```sh
$ systemctl status centrifugo.service 
● centrifugo.service - Centrifugo real-time messaging server
   Loaded: loaded (/lib/systemd/system/centrifugo.service; enabled; vendor preset: enabled)
   Active: active (running) since Пт 2018-04-20 10:17:19 +08; 1s ago
     Docs: https://github.com/centrifugal/centrifugo
  Process: 354 ExecStartPre=/usr/bin/centrifugo checkconfig -c $CONFIG (code=exited, status=0/SUCCESS)
...

$ dpkg -l | grep centrifug
ii  centrifugo                                  1.7.9-0                                      amd64        Real-time messaging server

$ sudo apt-get install --reinstall centrifugo
Чтение списков пакетов… Готово
Построение дерева зависимостей       
Чтение информации о состоянии… Готово
...
обновлено 0, установлено 0 новых пакетов, переустановлено 1 переустановлено, для удаления отмечено 0 пакетов, и 0 пакетов не обновлено.
Необходимо скачать 0 B/5 483 kB архивов.
После данной операции, объём занятого дискового пространства возрастёт на 0 B.
(Чтение базы данных … на данный момент установлено 261212 файлов и каталогов.)
Подготовка к распаковке …/centrifugo_1.7.9-0_amd64.deb …
Распаковывается centrifugo (1.7.9-0) на замену (1.7.9-0) …
Настраивается пакет centrifugo (1.7.9-0) …
Failed to execute operation: Too many levels of symbolic links
dpkg: ошибка при обработке пакета centrifugo (--configure):
 подпроцесс установлен сценарий post-installation возвратил код ошибки 1
При обработке следующих пакетов произошли ошибки:
 centrifugo
E: Sub-process /usr/bin/dpkg returned an error code (1)

$ dpkg -l | grep centrifug
iF  centrifugo                                  1.7.9-0                                      amd64        Real-time messaging server

$ systemctl status centrifugo.service 
● centrifugo.service - Centrifugo real-time messaging server
   Loaded: loaded (/lib/systemd/system/centrifugo.service; enabled; vendor preset: enabled)
   Active: active (running) since Пт 2018-04-20 10:17:19 +08; 3min 19s ago
     Docs: https://github.com/centrifugal/centrifugo
  Process: 354 ExecStartPre=/usr/bin/centrifugo checkconfig -c $CONFIG (code=exited, status=0/SUCCESS)
...
Warning: centrifugo.service changed on disk. Run 'systemctl daemon-reload' to reload units.
```

To fix package status I do something like so:
```sh
$ sudo systemctl disable centrifugo.service 
Removed symlink /etc/systemd/system/centrifugo.service.
Removed symlink /etc/systemd/system/multi-user.target.wants/centrifugo.service.

$ sudo apt-get install -f
Чтение списков пакетов… Готово
Построение дерева зависимостей       
Чтение информации о состоянии… Готово
...
обновлено 0, установлено 0 новых пакетов, для удаления отмечено 0 пакетов, и 0 пакетов не обновлено.
не установлено до конца или удалено 1 пакетов.
После данной операции, объём занятого дискового пространства возрастёт на 0 B.
Настраивается пакет centrifugo (1.7.9-0) …
Created symlink from /etc/systemd/system/centrifugo.service to /lib/systemd/system/centrifugo.service.
Created symlink from /etc/systemd/system/multi-user.target.wants/centrifugo.service to /lib/systemd/system/centrifugo.service.

$ systemctl status centrifugo.service 
● centrifugo.service - Centrifugo real-time messaging server
   Loaded: loaded (/lib/systemd/system/centrifugo.service; enabled; vendor preset: enabled)
   Active: active (running) since Пт 2018-04-20 10:17:19 +08; 19min ago
     Docs: https://github.com/centrifugal/centrifugo
...

$ dpkg -l | grep centrifug
ii  centrifugo                                  1.7.9-0                                      amd64        Real-time messaging server
```

**Comment by kaizer666** [comment]
I get the same error when updating

**Comment by FZambia** [comment]
@Vovan-VE hello, thanks for report, what is your Linux distribution/version?

**Comment by Vovan-VE** [comment]
Ubuntu 16.04

**Comment by FZambia** [comment]
Pushed a fix in cd3b73979b64032d6d311183f0d53679307cd110

**Comment by FZambia** [comment]
Released in [v1.8.0](https://github.com/centrifugal/centrifugo/releases/tag/v1.8.0)
